test/XyGovernanceProxy.test.js

```diff
@@ -4,25 +4,19 @@ const { Contracts, ZWeb3 } = require(`zos-lib`)
 ZWeb3.initialize(web3.currentProvider)
 
 const Governance = Contracts.getFromLocal(`XyGovernance`)
-const ERC20 = artifacts.require(`XyERC20Token.sol`)
```

Comment:
This XyERC20Token is modeled after XYO's token which unfortunately has an outdated transfer method.  That's why we use it in our project instead of the OZ standard.  It will not be deployed when we we deploy to main, and we'll sub in the official one on main net

Reply:
Sounds good. I think the proxy tests work with the OZ ERC20, but I'll be ready either way.